Documentation/LanguageServer/c_cpp_properties.json.md

```diff
@@ -45,43 +45,45 @@
   An array of configuration objects that provide the IntelliSense engine with information about your project and your preferences. By default, the extension creates a configuration for you based on your operating system. You may also add additional configurations.
 
 * #### `version`
-  We recommend you don't edit this field. It tracks the current version of the **c_cpp_properties.json** file so that the extension knows what properties and settings should be present and how to upgrade this file to the latest version.
+  We recommend you don't edit this field. It tracks the current version of the `c_cpp_properties.json` file so that the extension knows what properties and settings should be present and how to upgrade this file to the latest version.
 
 ## Configuration properties
 
 * #### `name`
-  A friendly name for the configuration. "Linux", "Mac", and "Win32" are special names that instruct the extension to load that configuration by default on the associated operating system unless additional configurations have been created. The status bar in VS Code will show you which configuration is active. You can also click on the label in the status bar to change the active configuration.
+  A friendly name that identifies a configuration. `Linux`, `Mac`, and `Win32` are special identifiers for configurations that will be auto-selected on those platforms, but the name of the identifier can be anything. The status bar in VS Code will show you which configuration is active. You can also click on the label in the status bar to change the active configuration.
```

Comment:
"the name of the identifier can be anything" is confusing. Maybe it should just be removed?